src/services/TemplateCaches.php

```diff
@@ -230,13 +230,17 @@ public function includeElementQueryInTemplateCaches(Event $event)
             $elementQuery->customFields = $customFields;
             $hash = md5($serialized);
 
+            gc_disable();
             foreach ($this->_cachedQueries as &$queries) {
                 $queries[$hash] = [
                     $elementQuery->elementType,
                     $serialized
                 ];
+                gc_collect_cycles();
             }
+            $queries = null;
```

Comment:
@blasvicco What’s the reason for this line, considering the `unset($queries)` on the next line?

Reply:
Thanks for the explanation! That was news to me.

Reply:
;)